src/crypto/tls/generate_cert.go

```diff
@@ -28,13 +29,14 @@ import (
 )
 
 var (
-	host       = flag.String("host", "", "Comma-separated hostnames and IPs to generate a certificate for")
-	validFrom  = flag.String("start-date", "", "Creation date formatted as Jan 1 15:04:05 2011")
-	validFor   = flag.Duration("duration", 365*24*time.Hour, "Duration that certificate is valid for")
-	isCA       = flag.Bool("ca", false, "whether this cert should be its own Certificate Authority")
-	rsaBits    = flag.Int("rsa-bits", 2048, "Size of RSA key to generate. Ignored if --ecdsa-curve is set")
-	ecdsaCurve = flag.String("ecdsa-curve", "", "ECDSA curve to use to generate a key. Valid values are P224, P256 (recommended), P384, P521")
-	ed25519Key = flag.Bool("ed25519", false, "Generate an Ed25519 key")
+	host            = flag.String("host", "", "Comma-separated hostnames and IPs to generate a certificate for")
+	validFrom       = flag.String("start-date", "", "Creation date formatted as Jan 1 15:04:05 2011")
+	validFor        = flag.Duration("duration", 365*24*time.Hour, "Duration that certificate is valid for")
+	isCA            = flag.Bool("ca", false, "whether this cert should be its own Certificate Authority")
+	rsaBits         = flag.Int("rsa-bits", 2048, "Size of RSA key to generate. Ignored if --ecdsa-curve is set")
+	ecdsaCurve      = flag.String("ecdsa-curve", "", "ECDSA curve to use to generate a key. Valid values are P224, P256 (recommended), P384, P521")
+	ed25519Key      = flag.Bool("ed25519", false, "Generate an Ed25519 key")
+	eddilithium3Key = flag.Bool("eddilithium3", false, "Generate an Ed25519-Dilithium3 key")
```

Comment:
While this is nice, it is difficult to see that it is 'ed25519 + dilithium', and might got confused if added with ed448...

Reply:
I changed the flag.